app/Repositories/Interfaces/BaseRepositoryInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Repositories\Interfaces;
+
+interface BaseRepositoryInterface
+{
+    /**
+     * Get all
+     * @return mixed
+     */
+    public function getAll();
+
+    /**
+     * Get one
+     * @param $id
+     * @return mixed
+     */
+    public function find($id);
+
+    /**
+     * Create
+     * @param array $attributes
+     * @return mixed
+     */
+    public function create(array $attributes);
+
+    /**
+     * Update
+     * @param $id
+     * @param array $attributes
+     * @return mixed
+     */
+    public function update($id, array $attributes);
+
+    /**
+     * Delete
+     * @param $id
+     * @return mixed
+     */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Interfaces/BaseRepositoryInterface.php</br>**Line**: 41</br>Expected 1 newline at end of file; 0 found